### PR TITLE
adds helpers functions used by the vertical scaling controllers

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
these functions will be used by https://github.com/openshift/cluster-etcd-operator/pull/768 and https://github.com/openshift/cluster-etcd-operator/pull/742

File: pkg/operator/ceohelpers/common.go
Comment (p0lyn0mial) [inline pkg/operator/ceohelpers/common.go:25]: @hasbro17  you have already reviewed this commit in https://github.com/openshift/cluster-etcd-operator/pull/767
I'll rebase once it merges

**Review by hasbro17 — APPROVED** [review]
/approve

I can tag lgtm once #767 merges, or if you prefer to keep it here, that's fine too.

**Pull request merged**
